src/prop/minisat/minisat.cpp

```diff
@@ -258,9 +258,9 @@ bool MinisatSatSolver::isDecision(SatVariable decn) const {
   return d_minisat->isDecision( decn );
 }
 
-int32_t MinisatSatSolver::getDecisionLevel(SatVariable v) const
+int32_t MinisatSatSolver::getSatContextLevel(SatVariable v) const
```

Comment:
I'm wondering if there is a better name for this, since this seems like it could be confused with the level of the cvc5::SatContext.  Would `getContextLevel` be a better name?